Unable to build container image using `buildpacks` build strategy when command not invoked from project base directory

### Describe the bug

When we build image using buildpacks build strategy, it works as long as command is run inside the project directory. If we run it from some different directory, build fails as `pack` CLI considers project directory as the one from which maven/gradle process got invoked.

To reproduce it you can clone https://github.com/rohankanojia-forks/jkube-buildpacks-poc-testing and run the following command from project base directory:
```
mvn -fmaven-it k8s:build
```

The build will fail with buildpacks complaining `No buildpack groups passed detection`

### Eclipse JKube version

SNAPSHOT

### Component

Kubernetes Maven Plugin

### Apache Maven version

other (please specify in additional context)

### Gradle version

None

### Steps to reproduce

1. Clone https://github.com/rohankanojia-forks/jkube-buildpacks-poc-testing
2. Run `mvn -fmaven-it k8s:build`

### Expected behavior

Buildpacks build should detect the configured directory via the build tool. It shouldn't assume project directory as the one from which build process is initiated.

JKube should pass the project using `--path` flag in `pack` CLI arguments

### Runtime

Kubernetes (vanilla)

### Kubernetes API Server version

1.25.3

### Environment

Linux

### Eclipse JKube Logs

```shell
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] Resolving plan... (try #10)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 provides unused native-image-application
[INFO] k8s: [detector] Resolving plan... (try #11)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-processed
[INFO] k8s: [detector] Resolving plan... (try #12)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/upx@3.4.3 provides unused upx
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 provides unused native-image-application
[INFO] k8s: [detector] Resolving plan... (try #13)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #14)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-processed
[INFO] k8s: [detector] Resolving plan... (try #15)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #16)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #17)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #18)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #19)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #20)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #21)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #22)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #23)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-processed
[INFO] k8s: [detector] Resolving plan... (try #24)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #25)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 requires jdk
[INFO] k8s: [detector] Resolving plan... (try #26)
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 requires jdk
[INFO] k8s: [detector] Resolving plan... (try #27)
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 requires jdk
[INFO] k8s: [detector] Resolving plan... (try #28)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #29)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-processed
[INFO] k8s: [detector] Resolving plan... (try #30)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #31)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 provides unused native-image-application
[INFO] k8s: [detector] Resolving plan... (try #32)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-processed
[INFO] k8s: [detector] Resolving plan... (try #33)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/upx@3.4.3 provides unused upx
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] Resolving plan... (try #34)
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 provides unused native-image-application
[INFO] k8s: [detector] Resolving plan... (try #35)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-processed
[INFO] k8s: [detector] Resolving plan... (try #36)
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 provides unused native-image-application
[INFO] k8s: [detector] Resolving plan... (try #37)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] Resolving plan... (try #38)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-processed
[INFO] k8s: [detector] Resolving plan... (try #39)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 provides unused native-image-application
[INFO] k8s: [detector] Resolving plan... (try #40)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 provides unused native-image-application
[INFO] k8s: [detector] Resolving plan... (try #41)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-processed
[INFO] k8s: [detector] Resolving plan... (try #42)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/upx@3.4.3 provides unused upx
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 provides unused native-image-application
[INFO] k8s: [detector] Resolving plan... (try #43)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #44)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #45)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #46)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #47)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #48)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #49)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #50)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #51)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #52)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #53)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-processed
[INFO] k8s: [detector] Resolving plan... (try #54)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #55)
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 requires jdk
[INFO] k8s: [detector] Resolving plan... (try #56)
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 requires jdk
[INFO] k8s: [detector] Resolving plan... (try #57)
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 requires jdk
[INFO] k8s: [detector] Resolving plan... (try #58)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #59)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #60)
[INFO] k8s: [detector] fail: paketo-buildpacks/native-image@5.12.1 requires native-image-builder
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/leiningen@4.6.4 ========
[INFO] k8s: [detector] SKIPPED: project.clj could not be found in /workspace/project.clj
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/clojure-tools@2.8.5 ========
[INFO] k8s: [detector] SKIPPED: no 'deps.edn' file found in the application path
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/gradle@7.3.0 ========
[INFO] k8s: [detector]   Build Configuration:
[INFO] k8s: [detector]     $BP_EXCLUDE_FILES                                                                       colon separated list of glob patterns, matched source files are removed
[INFO] k8s: [detector]     $BP_GRADLE_ADDITIONAL_BUILD_ARGUMENTS                                                   the additionnal arguments (appended to BP_GRADLE_BUILD_ARGUMENTS) to pass to Gradle
[INFO] k8s: [detector]     $BP_GRADLE_BUILD_ARGUMENTS             --no-daemon -Dorg.gradle.welcome=never assemble  the arguments to pass to Gradle
[INFO] k8s: [detector]     $BP_GRADLE_BUILD_FILE                                                                   the location of the main build config file, relative to the application root
[INFO] k8s: [detector]     $BP_GRADLE_BUILT_ARTIFACT              build/libs/*.[jw]ar                              the built application artifact explicitly.  Supersedes $BP_GRADLE_BUILT_MODULE
[INFO] k8s: [detector]     $BP_GRADLE_BUILT_MODULE                                                                 the module to find application artifact in
[INFO] k8s: [detector]     $BP_GRADLE_INIT_SCRIPT_PATH                                                             the path to a Gradle init script file
[INFO] k8s: [detector]     $BP_INCLUDE_FILES                                                                       colon separated list of glob patterns, matched source files are included
[INFO] k8s: [detector]     $BP_JAVA_INSTALL_NODE                  false                                            whether to install Yarn/Node binaries based on the presence of a package.json or yarn.lock file
[INFO] k8s: [detector]     $BP_NODE_PROJECT_PATH                                                                   configure a project subdirectory to look for `package.json` and `yarn.lock` files
[INFO] k8s: [detector] SKIPPED: No plans could be resolved
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/sbt@6.12.2 ========
[INFO] k8s: [detector] SKIPPED: build.sbt could not be found in /workspace/build.sbt
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/executable-jar@6.7.4 ========
[INFO] k8s: [detector]   Build Configuration:
[INFO] k8s: [detector]     $BP_LIVE_RELOAD_ENABLED  false  enable live process reload in the image
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/apache-tomcat@7.13.7 ========
[INFO] k8s: [detector]   Build Configuration:
[INFO] k8s: [detector]     $BP_JAVA_APP_SERVER                             the application server to use
[INFO] k8s: [detector]     $BP_TOMCAT_CONTEXT_PATH                         the application context path
[INFO] k8s: [detector]     $BP_TOMCAT_ENV_PROPERTY_SOURCE_DISABLED  false  Disable Tomcat's EnvironmentPropertySource
[INFO] k8s: [detector]     $BP_TOMCAT_EXT_CONF_SHA256                      the SHA256 hash of the external Tomcat configuration archive
[INFO] k8s: [detector]     $BP_TOMCAT_EXT_CONF_STRIP                0      the number of directory components to strip from the external Tomcat configuration archive
[INFO] k8s: [detector]     $BP_TOMCAT_EXT_CONF_URI                         the download location of the external Tomcat configuration
[INFO] k8s: [detector]     $BP_TOMCAT_EXT_CONF_VERSION                     the version of the external Tomcat configuration
[INFO] k8s: [detector]     $BP_TOMCAT_VERSION                       9.*    the Tomcat version
[INFO] k8s: [detector]   Launch Configuration:
[INFO] k8s: [detector]     $BPL_TOMCAT_ACCESS_LOGGING_ENABLED              the Tomcat access logging state
[INFO] k8s: [detector] PASSED: a WEB-INF directory was not found, this is normal when building from source
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/apache-tomee@1.7.4 ========
[INFO] k8s: [detector]   Build Configuration:
[INFO] k8s: [detector]     $BP_JAVA_APP_SERVER                                   the application server to use
[INFO] k8s: [detector]     $BP_TOMEE_CONTEXT_PATH                                the application context path
[INFO] k8s: [detector]     $BP_TOMEE_DISTRIBUTION                  microprofile  the Tomee distribution
[INFO] k8s: [detector]     $BP_TOMEE_ENV_PROPERTY_SOURCE_DISABLED  false         Disable Tomcat's EnvironmentPropertySource
[INFO] k8s: [detector]     $BP_TOMEE_EXT_CONF_SHA256                             the SHA256 hash of the external Tomee configuration archive
[INFO] k8s: [detector]     $BP_TOMEE_EXT_CONF_STRIP                0             the number of directory components to strip from the external Tomee configuration archive
[INFO] k8s: [detector]     $BP_TOMEE_EXT_CONF_URI                                the download location of the external Tomee configuration
[INFO] k8s: [detector]     $BP_TOMEE_EXT_CONF_VERSION                            the version of the external Tomee configuration
[INFO] k8s: [detector]     $BP_TOMEE_VERSION                       8.*           the Tomee version
[INFO] k8s: [detector]   Launch Configuration:
[INFO] k8s: [detector]     $BPL_TOMEE_ACCESS_LOGGING_ENABLED                     the Tomee access logging state
[INFO] k8s: [detector] PASSED: a WEB-INF directory was not found, this is normal when building from source
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/procfile@5.6.4 ========
[INFO] k8s: [detector] SKIPPED: No procfile found from either source path or binding.
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/jattach@1.4.4 ========
[INFO] k8s: [detector] SKIPPED: BP_JATTACH_ENABLED was not set to true
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/azure-application-insights@5.15.1 ========
[INFO] k8s: [detector] SKIPPED: no binding of type ApplicationInsights found
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/google-stackdriver@7.6.1 ========
[INFO] k8s: [detector] SKIPPED: no bindings of type 'StackdriverDebugger' or 'StackdriverProfiler' found
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/datadog@3.15.0 ========
[INFO] k8s: [detector]   Build Configuration:
[INFO] k8s: [detector]     $BP_DATADOG_ENABLED    false  whether to contribute the Datadog trace agent
[INFO] k8s: [detector]   Launch Configuration:
[INFO] k8s: [detector]     $BPL_DATADOG_DISABLED  false  whether to disable the Datadog trace agent (Java only!)
[INFO] k8s: [detector] SKIPPED: variable 'BP_DATADOG_ENABLED' not set to true
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/java-memory-assistant@1.4.4 ========
[INFO] k8s: [detector] SKIPPED: BP_JMA_ENABLED was not set to true
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/image-labels@4.5.2 ========
[INFO] k8s: [detector]   Build Configuration:
[INFO] k8s: [detector]     $BP_IMAGE_LABELS       arbitrary image labels
[INFO] k8s: [detector]     $BP_OCI_AUTHORS        the org.opencontainers.image.authors image label
[INFO] k8s: [detector]     $BP_OCI_CREATED        the org.opencontainers.image.created image label
[INFO] k8s: [detector]     $BP_OCI_DESCRIPTION    the org.opencontainers.image.description image label
[INFO] k8s: [detector]     $BP_OCI_DOCUMENTATION  the org.opencontainers.image.documentation image label
[INFO] k8s: [detector]     $BP_OCI_LICENSES       the org.opencontainers.image.licenses image label
[INFO] k8s: [detector]     $BP_OCI_REF_NAME       the org.opencontainers.image.ref.name image label
[INFO] k8s: [detector]     $BP_OCI_REVISION       the org.opencontainers.image.revision image label
[INFO] k8s: [detector]     $BP_OCI_SOURCE         the org.opencontainers.image.source image label
[INFO] k8s: [detector]     $BP_OCI_TITLE          the org.opencontainers.image.title image label
[INFO] k8s: [detector]     $BP_OCI_URL            the org.opencontainers.image.url image label
[INFO] k8s: [detector]     $BP_OCI_VENDOR         the org.opencontainers.image.vendor image label
[INFO] k8s: [detector]     $BP_OCI_VERSION        the org.opencontainers.image.version image label
[INFO] k8s: [detector] SKIPPED: No supported environment variables were set
[INFO] k8s: [detector] ======== Results ========
[INFO] k8s: [detector] pass: paketo-buildpacks/ca-certificates@3.6.3
[INFO] k8s: [detector] pass: paketo-buildpacks/bellsoft-liberica@10.2.6
[INFO] k8s: [detector] pass: paketo-buildpacks/yarn@1.1.9
[INFO] k8s: [detector] pass: paketo-buildpacks/node-engine@1.6.0
[INFO] k8s: [detector] pass: paketo-buildpacks/syft@1.32.1
[INFO] k8s: [detector] skip: paketo-buildpacks/leiningen@4.6.4
[INFO] k8s: [detector] skip: paketo-buildpacks/clojure-tools@2.8.5
[INFO] k8s: [detector] skip: paketo-buildpacks/gradle@7.3.0
[INFO] k8s: [detector] pass: paketo-buildpacks/maven@6.15.5
[INFO] k8s: [detector] skip: paketo-buildpacks/sbt@6.12.2
[INFO] k8s: [detector] pass: paketo-buildpacks/watchexec@2.8.3
[INFO] k8s: [detector] pass: paketo-buildpacks/executable-jar@6.7.4
[INFO] k8s: [detector] pass: paketo-buildpacks/apache-tomcat@7.13.7
[INFO] k8s: [detector] pass: paketo-buildpacks/apache-tomee@1.7.4
[INFO] k8s: [detector] pass: paketo-buildpacks/liberty@3.8.2
[INFO] k8s: [detector] pass: paketo-buildpacks/dist-zip@5.6.4
[INFO] k8s: [detector] pass: paketo-buildpacks/spring-boot@5.26.1
[INFO] k8s: [detector] skip: paketo-buildpacks/procfile@5.6.4
[INFO] k8s: [detector] skip: paketo-buildpacks/jattach@1.4.4
[INFO] k8s: [detector] skip: paketo-buildpacks/azure-application-insights@5.15.1
[INFO] k8s: [detector] skip: paketo-buildpacks/google-stackdriver@7.6.1
[INFO] k8s: [detector] skip: paketo-buildpacks/datadog@3.15.0
[INFO] k8s: [detector] skip: paketo-buildpacks/java-memory-assistant@1.4.4
[INFO] k8s: [detector] skip: paketo-buildpacks/encrypt-at-rest@4.5.5
[INFO] k8s: [detector] skip: paketo-buildpacks/environment-variables@4.5.3
[INFO] k8s: [detector] skip: paketo-buildpacks/image-labels@4.5.2
[INFO] k8s: [detector] Resolving plan... (try #1)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] Resolving plan... (try #2)
[INFO] k8s: [detector] skip: paketo-buildpacks/watchexec@2.8.3 provides unused watchexec
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] fail: paketo-buildpacks/ca-certificates@3.6.3 provides unused ca-certificates
[INFO] k8s: [detector] Resolving plan... (try #3)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused npm
[INFO] k8s: [detector] fail: paketo-buildpacks/ca-certificates@3.6.3 provides unused ca-certificates
[INFO] k8s: [detector] Resolving plan... (try #4)
[INFO] k8s: [detector] fail: paketo-buildpacks/ca-certificates@3.6.3 provides unused ca-certificates
[INFO] k8s: [detector] Resolving plan... (try #5)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] fail: paketo-buildpacks/bellsoft-liberica@10.2.6 provides unused native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #6)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] fail: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] Resolving plan... (try #7)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] fail: paketo-buildpacks/ca-certificates@3.6.3 provides unused ca-certificates
[INFO] k8s: [detector] Resolving plan... (try #8)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/bellsoft-liberica@10.2.6 provides unused native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #9)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] Resolving plan... (try #10)
[INFO] k8s: [detector] skip: paketo-buildpacks/watchexec@2.8.3 provides unused watchexec
[INFO] k8s: [detector] fail: paketo-buildpacks/ca-certificates@3.6.3 provides unused ca-certificates
[INFO] k8s: [detector] Resolving plan... (try #11)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] skip: paketo-buildpacks/watchexec@2.8.3 provides unused watchexec
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] Resolving plan... (try #12)
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] fail: paketo-buildpacks/ca-certificates@3.6.3 provides unused ca-certificates
[INFO] k8s: [detector] Resolving plan... (try #13)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] skip: paketo-buildpacks/watchexec@2.8.3 provides unused watchexec
[INFO] k8s: [detector] fail: paketo-buildpacks/ca-certificates@3.6.3 provides unused ca-certificates
[INFO] k8s: [detector] Resolving plan... (try #14)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] skip: paketo-buildpacks/watchexec@2.8.3 provides unused watchexec
[INFO] k8s: [detector] fail: paketo-buildpacks/ca-certificates@3.6.3 provides unused ca-certificates
[INFO] k8s: [detector] Resolving plan... (try #15)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/watchexec@2.8.3 provides unused watchexec
[INFO] k8s: [detector] fail: paketo-buildpacks/ca-certificates@3.6.3 provides unused ca-certificates
[INFO] k8s: [detector] Resolving plan... (try #16)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/ca-certificates@3.6.3 provides unused ca-certificates
[INFO] k8s: [detector] Resolving plan... (try #17)
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 requires jdk
[INFO] k8s: [detector] Resolving plan... (try #18)
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jdk
[INFO] k8s: [detector] fail: paketo-buildpacks/ca-certificates@3.6.3 provides unused ca-certificates
[INFO] k8s: [detector] Resolving plan... (try #19)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 requires jdk
[INFO] k8s: [detector] Resolving plan... (try #20)
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jdk
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused npm
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] fail: paketo-buildpacks/ca-certificates@3.6.3 provides unused ca-certificates
[INFO] k8s: [detector] Resolving plan... (try #21)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/bellsoft-liberica@10.2.6 provides unused native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #22)
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] skip: paketo-buildpacks/watchexec@2.8.3 provides unused watchexec
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] fail: paketo-buildpacks/bellsoft-liberica@10.2.6 provides unused native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #23)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] Resolving plan... (try #24)
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] fail: paketo-buildpacks/bellsoft-liberica@10.2.6 provides unused native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #25)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] Resolving plan... (try #26)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/bellsoft-liberica@10.2.6 provides unused native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #27)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] skip: paketo-buildpacks/watchexec@2.8.3 provides unused watchexec
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused npm
[INFO] k8s: [detector] fail: paketo-buildpacks/bellsoft-liberica@10.2.6 provides unused native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #28)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] fail: paketo-buildpacks/bellsoft-liberica@10.2.6 provides unused native-image-builder
[INFO] k8s: [detector] Resolving plan... (try #29)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] Resolving plan... (try #30)
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] fail: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] Resolving plan... (try #31)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] Resolving plan... (try #32)
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] skip: paketo-buildpacks/watchexec@2.8.3 provides unused watchexec
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused npm
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/spring-boot@5.26.1 requires jvm-application
[INFO] k8s: [detector] fail: paketo-buildpacks/bellsoft-liberica@10.2.6 provides unused jdk
[INFO] k8s: [detector] Resolving plan... (try #33)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] fail: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] Resolving plan... (try #34)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] fail: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] Resolving plan... (try #35)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] skip: paketo-buildpacks/watchexec@2.8.3 provides unused watchexec
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused npm
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] Resolving plan... (try #36)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jre
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused npm
[INFO] k8s: [detector] skip: paketo-buildpacks/watchexec@2.8.3 provides unused watchexec
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] skip: paketo-buildpacks/spring-boot@5.26.1 requires jvm-application
[INFO] k8s: [detector] fail: paketo-buildpacks/bellsoft-liberica@10.2.6 provides unused jdk
[INFO] k8s: [detector] Resolving plan... (try #37)
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 requires jdk
[INFO] k8s: [detector] Resolving plan... (try #38)
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jdk
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] fail: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] Resolving plan... (try #39)
[INFO] k8s: [detector] fail: paketo-buildpacks/maven@6.15.5 requires jdk
[INFO] k8s: [detector] Resolving plan... (try #40)
[INFO] k8s: [detector] skip: paketo-buildpacks/liberty@3.8.2 requires jdk
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn@1.1.9 provides unused yarn
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused npm
[INFO] k8s: [detector] skip: paketo-buildpacks/watchexec@2.8.3 provides unused watchexec
[INFO] k8s: [detector] skip: paketo-buildpacks/node-engine@1.6.0 provides unused node
[INFO] k8s: [detector] skip: paketo-buildpacks/maven@6.15.5 provides unused maven
[INFO] k8s: [detector] skip: paketo-buildpacks/executable-jar@6.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomcat@7.13.7 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/apache-tomee@1.7.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/dist-zip@5.6.4 requires jvm-application-package
[INFO] k8s: [detector] skip: paketo-buildpacks/spring-boot@5.26.1 requires jvm-application
[INFO] k8s: [detector] fail: paketo-buildpacks/bellsoft-liberica@10.2.6 provides unused jre
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/yarn-install@1.1.3 ========
[INFO] k8s: [detector] no 'yarn.lock' file found in the project path /workspace
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/node-run-script@1.0.10 ========
[INFO] k8s: [detector] no package.json file present
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/node-start@1.0.11 ========
[INFO] k8s: [detector] could not find app in /workspace: expected one of server.js | app.js | main.js | index.js
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/yarn-start@1.0.11 ========
[INFO] k8s: [detector] no 'yarn.lock' found in the project path /workspace
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/procfile@5.6.3 ========
[INFO] k8s: [detector] SKIPPED: No procfile found from either source path or binding.
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/image-labels@4.5.1 ========
[INFO] k8s: [detector]   Build Configuration:
[INFO] k8s: [detector]     $BP_IMAGE_LABELS       arbitrary image labels
[INFO] k8s: [detector]     $BP_OCI_AUTHORS        the org.opencontainers.image.authors image label
[INFO] k8s: [detector]     $BP_OCI_CREATED        the org.opencontainers.image.created image label
[INFO] k8s: [detector]     $BP_OCI_DESCRIPTION    the org.opencontainers.image.description image label
[INFO] k8s: [detector]     $BP_OCI_DOCUMENTATION  the org.opencontainers.image.documentation image label
[INFO] k8s: [detector]     $BP_OCI_LICENSES       the org.opencontainers.image.licenses image label
[INFO] k8s: [detector]     $BP_OCI_REF_NAME       the org.opencontainers.image.ref.name image label
[INFO] k8s: [detector]     $BP_OCI_REVISION       the org.opencontainers.image.revision image label
[INFO] k8s: [detector]     $BP_OCI_SOURCE         the org.opencontainers.image.source image label
[INFO] k8s: [detector]     $BP_OCI_TITLE          the org.opencontainers.image.title image label
[INFO] k8s: [detector]     $BP_OCI_URL            the org.opencontainers.image.url image label
[INFO] k8s: [detector]     $BP_OCI_VENDOR         the org.opencontainers.image.vendor image label
[INFO] k8s: [detector]     $BP_OCI_VERSION        the org.opencontainers.image.version image label
[INFO] k8s: [detector] SKIPPED: No supported environment variables were set
[INFO] k8s: [detector] ======== Results ========
[INFO] k8s: [detector] pass: paketo-buildpacks/ca-certificates@3.6.2
[INFO] k8s: [detector] pass: paketo-buildpacks/watchexec@2.8.2
[INFO] k8s: [detector] pass: paketo-buildpacks/node-engine@1.6.0
[INFO] k8s: [detector] pass: paketo-buildpacks/yarn@1.1.9
[INFO] k8s: [detector] fail: paketo-buildpacks/yarn-install@1.1.3
[INFO] k8s: [detector] skip: paketo-buildpacks/node-run-script@1.0.10
[INFO] k8s: [detector] skip: paketo-buildpacks/node-start@1.0.11
[INFO] k8s: [detector] skip: paketo-buildpacks/yarn-start@1.0.11
[INFO] k8s: [detector] skip: paketo-buildpacks/procfile@5.6.3
[INFO] k8s: [detector] skip: paketo-buildpacks/datadog@3.6.0
[INFO] k8s: [detector] skip: paketo-buildpacks/environment-variables@4.5.2
[INFO] k8s: [detector] skip: paketo-buildpacks/image-labels@4.5.1
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/npm-install@1.1.4 ========
[INFO] k8s: [detector] no 'package.json' found in project path /workspace
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/node-run-script@1.0.10 ========
[INFO] k8s: [detector] no package.json file present
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/node-start@1.0.11 ========
[INFO] k8s: [detector] could not find app in /workspace: expected one of server.js | app.js | main.js | index.js
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/npm-start@1.0.11 ========
[INFO] k8s: [detector] no 'package.json' found in project path /workspace
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/procfile@5.6.3 ========
[INFO] k8s: [detector] SKIPPED: No procfile found from either source path or binding.
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/image-labels@4.5.1 ========
[INFO] k8s: [detector]   Build Configuration:
[INFO] k8s: [detector]     $BP_IMAGE_LABELS       arbitrary image labels
[INFO] k8s: [detector]     $BP_OCI_AUTHORS        the org.opencontainers.image.authors image label
[INFO] k8s: [detector]     $BP_OCI_CREATED        the org.opencontainers.image.created image label
[INFO] k8s: [detector]     $BP_OCI_DESCRIPTION    the org.opencontainers.image.description image label
[INFO] k8s: [detector]     $BP_OCI_DOCUMENTATION  the org.opencontainers.image.documentation image label
[INFO] k8s: [detector]     $BP_OCI_LICENSES       the org.opencontainers.image.licenses image label
[INFO] k8s: [detector]     $BP_OCI_REF_NAME       the org.opencontainers.image.ref.name image label
[INFO] k8s: [detector]     $BP_OCI_REVISION       the org.opencontainers.image.revision image label
[INFO] k8s: [detector]     $BP_OCI_SOURCE         the org.opencontainers.image.source image label
[INFO] k8s: [detector]     $BP_OCI_TITLE          the org.opencontainers.image.title image label
[INFO] k8s: [detector]     $BP_OCI_URL            the org.opencontainers.image.url image label
[INFO] k8s: [detector]     $BP_OCI_VENDOR         the org.opencontainers.image.vendor image label
[INFO] k8s: [detector]     $BP_OCI_VERSION        the org.opencontainers.image.version image label
[INFO] k8s: [detector] SKIPPED: No supported environment variables were set
[INFO] k8s: [detector] ======== Results ========
[INFO] k8s: [detector] pass: paketo-buildpacks/ca-certificates@3.6.2
[INFO] k8s: [detector] pass: paketo-buildpacks/watchexec@2.8.2
[INFO] k8s: [detector] pass: paketo-buildpacks/node-engine@1.6.0
[INFO] k8s: [detector] fail: paketo-buildpacks/npm-install@1.1.4
[INFO] k8s: [detector] skip: paketo-buildpacks/node-run-script@1.0.10
[INFO] k8s: [detector] skip: paketo-buildpacks/node-start@1.0.11
[INFO] k8s: [detector] skip: paketo-buildpacks/npm-start@1.0.11
[INFO] k8s: [detector] skip: paketo-buildpacks/procfile@5.6.3
[INFO] k8s: [detector] skip: paketo-buildpacks/datadog@3.6.0
[INFO] k8s: [detector] skip: paketo-buildpacks/environment-variables@4.5.2
[INFO] k8s: [detector] skip: paketo-buildpacks/image-labels@4.5.1
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/node-start@1.0.11 ========
[INFO] k8s: [detector] could not find app in /workspace: expected one of server.js | app.js | main.js | index.js
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/procfile@5.6.3 ========
[INFO] k8s: [detector] SKIPPED: No procfile found from either source path or binding.
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/image-labels@4.5.1 ========
[INFO] k8s: [detector]   Build Configuration:
[INFO] k8s: [detector]     $BP_IMAGE_LABELS       arbitrary image labels
[INFO] k8s: [detector]     $BP_OCI_AUTHORS        the org.opencontainers.image.authors image label
[INFO] k8s: [detector]     $BP_OCI_CREATED        the org.opencontainers.image.created image label
[INFO] k8s: [detector]     $BP_OCI_DESCRIPTION    the org.opencontainers.image.description image label
[INFO] k8s: [detector]     $BP_OCI_DOCUMENTATION  the org.opencontainers.image.documentation image label
[INFO] k8s: [detector]     $BP_OCI_LICENSES       the org.opencontainers.image.licenses image label
[INFO] k8s: [detector]     $BP_OCI_REF_NAME       the org.opencontainers.image.ref.name image label
[INFO] k8s: [detector]     $BP_OCI_REVISION       the org.opencontainers.image.revision image label
[INFO] k8s: [detector]     $BP_OCI_SOURCE         the org.opencontainers.image.source image label
[INFO] k8s: [detector]     $BP_OCI_TITLE          the org.opencontainers.image.title image label
[INFO] k8s: [detector]     $BP_OCI_URL            the org.opencontainers.image.url image label
[INFO] k8s: [detector]     $BP_OCI_VENDOR         the org.opencontainers.image.vendor image label
[INFO] k8s: [detector]     $BP_OCI_VERSION        the org.opencontainers.image.version image label
[INFO] k8s: [detector] SKIPPED: No supported environment variables were set
[INFO] k8s: [detector] ======== Results ========
[INFO] k8s: [detector] pass: paketo-buildpacks/ca-certificates@3.6.2
[INFO] k8s: [detector] pass: paketo-buildpacks/watchexec@2.8.2
[INFO] k8s: [detector] pass: paketo-buildpacks/node-engine@1.6.0
[INFO] k8s: [detector] fail: paketo-buildpacks/node-start@1.0.11
[INFO] k8s: [detector] skip: paketo-buildpacks/procfile@5.6.3
[INFO] k8s: [detector] skip: paketo-buildpacks/datadog@3.6.0
[INFO] k8s: [detector] skip: paketo-buildpacks/environment-variables@4.5.2
[INFO] k8s: [detector] skip: paketo-buildpacks/image-labels@4.5.1
[INFO] k8s: [detector] ======== Output: paketo-buildpacks/procfile@5.6.4 ========
[INFO] k8s: [detector] SKIPPED: No procfile found from either source path or binding.
[INFO] k8s: [detector] ======== Results ========
[INFO] k8s: [detector] fail: paketo-buildpacks/procfile@5.6.4
[INFO] k8s: [detector] ERROR: No buildpack groups passed detection.
[INFO] k8s: [detector] ERROR: Please check that you are running against the correct path.
[INFO] k8s: [detector] ERROR: failed to detect: no buildpacks participating
[ERROR] k8s: Failed to execute the build [Process Existed With : 1 [Process '/home/rokumar/.jkube/pack build rohankanojiabot/spring-boot3-demo-jkube:latest --builder paketobuildpacks/builder:base --creation-time now --tag rohankanojiabot/spring-boot3-demo-jkube:latest --env JAVA_MAIN_CLASS=org.springframework.boot.loader.JarLauncher --env JAVA_APP_DIR=/deployments' exited with status 1]]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  12.612 s
[INFO] Finished at: 2024-05-31T17:27:36+05:30
[INFO] ------------------------------------------------------------------------
```


### Sample Reproducer Project

https://github.com/rohankanojia-forks/jkube-buildpacks-poc-testing

### Additional context

_No response_

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -35,6 +35,7 @@ Usage:
 * Fix #3007: Kubernetes Maven Plugin generating resource manifests with line feeds on Windows
 * Fix #3067: Helm Push uses configured docker global and push registries instead of pull 
 * Fix #2110: Add new helm dependency update goal task (`k8s:helm-dependency-update` for maven and `k8sHelmDependencyUpdate` for gradle)
+* Fix #3122: JKube should also pass project directory in `buildpacks` build strategy
 
 ### 1.16.2 (2024-03-27)
 * Fix #2461: `k8s:watch`/`k8sWatch` should throw error in `buildpacks` build strategy

--- a/jkube-kit/build/service/buildpacks/src/main/java/org/eclipse/jkube/kit/service/buildpacks/BuildPackBuildOptions.java
+++ b/jkube-kit/build/service/buildpacks/src/main/java/org/eclipse/jkube/kit/service/buildpacks/BuildPackBuildOptions.java
@@ -38,4 +38,5 @@ public class BuildPackBuildOptions {
   private List<String> tags;
   private Map<String, String> env;
   private List<String> volumes;
+  private String path;
 }

--- a/jkube-kit/build/service/buildpacks/src/main/java/org/eclipse/jkube/kit/service/buildpacks/controller/BuildPackCliController.java
+++ b/jkube-kit/build/service/buildpacks/src/main/java/org/eclipse/jkube/kit/service/buildpacks/controller/BuildPackCliController.java
@@ -89,6 +89,7 @@ public class BuildPackCliController implements BuildPackController {
     if (buildOptions.isClearCache()) {
       buildArgs.add("--clear-cache");
     }
+    buildArgs.addAll(Arrays.asList("--path", buildOptions.getPath()));
     return buildArgs;
   }
 

--- a/jkube-kit/build/service/buildpacks/src/test/java/org/eclipse/jkube/kit/service/buildpacks/BuildPackCliControllerTest.java
+++ b/jkube-kit/build/service/buildpacks/src/test/java/org/eclipse/jkube/kit/service/buildpacks/BuildPackCliControllerTest.java
@@ -20,6 +20,7 @@ import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
 import java.io.File;
 import java.util.Arrays;
@@ -37,6 +38,9 @@ class BuildPackCliControllerTest {
   private BuildPackBuildOptions buildOptions;
   private File pack;
 
+  @TempDir
+  private File temporaryFolder;
+
   @BeforeEach
   void setUp() {
     kitLogger = spy(new KitLogger.SilentLogger());
@@ -47,6 +51,7 @@ class BuildPackCliControllerTest {
         .imageName("foo/bar:latest")
         .builderImage("foo/builder:base")
         .creationTime("now")
+        .path(temporaryFolder.getAbsolutePath())
         .build();
   }
 
@@ -60,7 +65,7 @@ class BuildPackCliControllerTest {
       buildPackCliController.build(buildOptions);
 
       // Then
-      verify(kitLogger).info("[[s]]%s", "build foo/bar:latest --builder foo/builder:base --creation-time now");
+      verify(kitLogger).info("[[s]]%s", "build foo/bar:latest --builder foo/builder:base --creation-time now --path " + temporaryFolder.getAbsolutePath());
     }
 
     @Test
@@ -77,7 +82,7 @@ class BuildPackCliControllerTest {
       // When
       buildPackCliController.build(buildOptions);
       // Then
-      verify(kitLogger).info("[[s]]%s", "build foo/bar:latest --builder foo/builder:base --creation-time now --pull-policy if-not-present --volume /tmp/volume:/platform/volume:ro --tag foo/bar:t1 --tag foo/bar:t2 --tag foo/bar:t3 --env BP_SPRING_CLOUD_BINDINGS_DISABLED=true --clear-cache");
+      verify(kitLogger).info("[[s]]%s", "build foo/bar:latest --builder foo/builder:base --creation-time now --pull-policy if-not-present --volume /tmp/volume:/platform/volume:ro --tag foo/bar:t1 --tag foo/bar:t2 --tag foo/bar:t3 --env BP_SPRING_CLOUD_BINDINGS_DISABLED=true --clear-cache --path " + temporaryFolder.getAbsolutePath());
     }
 
     @Test

--- a/jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/kubernetes/BuildPackBuildService.java
+++ b/jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/kubernetes/BuildPackBuildService.java
@@ -80,6 +80,7 @@ public class BuildPackBuildService extends AbstractImageBuildService {
     BuildPackBuildOptions.BuildPackBuildOptionsBuilder buildPackBuildOptionsBuilder = BuildPackBuildOptions.builder()
         .imageName(imageConfiguration.getName())
         .builderImage(builderImage)
+        .path(jKubeConfiguration.getBasedir().getAbsolutePath())
         .creationTime("now");
     if (imageConfiguration.getBuild() != null) {
       if (StringUtils.isNotBlank(imageConfiguration.getBuild().getImagePullPolicy())) {

--- a/jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/kubernetes/BuildPackBuildServiceTest.java
+++ b/jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/kubernetes/BuildPackBuildServiceTest.java
@@ -17,6 +17,7 @@ import org.eclipse.jkube.kit.build.service.docker.DockerServiceHub;
 import org.eclipse.jkube.kit.build.service.docker.access.DockerAccess;
 import org.eclipse.jkube.kit.build.service.docker.access.DockerAccessException;
 import org.eclipse.jkube.kit.common.JKubeConfiguration;
+import org.eclipse.jkube.kit.common.JavaProject;
 import org.eclipse.jkube.kit.common.KitLogger;
 import org.eclipse.jkube.kit.common.RegistryConfig;
 import org.eclipse.jkube.kit.common.TestHttpBuildPacksArtifactsServer;
@@ -79,7 +80,11 @@ class BuildPackBuildServiceTest {
         .platformMode(RuntimeMode.KUBERNETES)
         .dockerServiceHub(mock(DockerServiceHub.class))
         .buildServiceConfig(buildServiceConfig)
-        .configuration(JKubeConfiguration.builder().build())
+        .configuration(JKubeConfiguration.builder()
+          .project(JavaProject.builder()
+            .baseDirectory(temporaryFolder)
+            .build())
+          .build())
         .build();
     imageConfiguration = ImageConfiguration.builder()
         .name("foo/bar:latest")
@@ -170,7 +175,7 @@ class BuildPackBuildServiceTest {
         buildPackBuildService.buildSingleImage(imageConfiguration);
 
         // Then
-        verify(kitLogger).info("[[s]]%s","build foo/bar:latest --builder cnbs/sample-builder:bionic --creation-time now");
+        verify(kitLogger).info("[[s]]%s","build foo/bar:latest --builder cnbs/sample-builder:bionic --creation-time now --path " + temporaryFolder.getAbsolutePath());
       }
 
       @Test
@@ -183,7 +188,7 @@ class BuildPackBuildServiceTest {
         buildPackBuildService.buildSingleImage(imageConfiguration);
 
         // Then
-        verify(kitLogger).info("[[s]]%s","build foo/bar:latest --builder paketobuildpacks/builder:base --creation-time now");
+        verify(kitLogger).info("[[s]]%s","build foo/bar:latest --builder paketobuildpacks/builder:base --creation-time now --path " + temporaryFolder.getAbsolutePath());
       }
     }
 
@@ -197,7 +202,7 @@ class BuildPackBuildServiceTest {
         buildPackBuildService.buildSingleImage(imageConfiguration);
 
         // Then
-        verify(kitLogger).info("[[s]]%s", "build foo/bar:latest --builder paketobuildpacks/builder:base --creation-time now");
+        verify(kitLogger).info("[[s]]%s", "build foo/bar:latest --builder paketobuildpacks/builder:base --creation-time now --path " + temporaryFolder.getAbsolutePath());
       }
 
       @Test
@@ -219,7 +224,7 @@ class BuildPackBuildServiceTest {
         buildPackBuildService.buildSingleImage(imageConfiguration);
 
         // Then
-        verify(kitLogger).info("[[s]]%s", "build foo/bar:latest --builder paketobuildpacks/builder:tiny --creation-time now --pull-policy if-not-present --volume /tmp/volume:/platform/volume:ro --tag foo/bar:t1 --tag foo/bar:t2 --tag foo/bar:t3 --env BP_SPRING_CLOUD_BINDINGS_DISABLED=true --clear-cache");
+        verify(kitLogger).info("[[s]]%s", "build foo/bar:latest --builder paketobuildpacks/builder:tiny --creation-time now --pull-policy if-not-present --volume /tmp/volume:/platform/volume:ro --tag foo/bar:t1 --tag foo/bar:t2 --tag foo/bar:t3 --env BP_SPRING_CLOUD_BINDINGS_DISABLED=true --clear-cache --path " + temporaryFolder.getAbsolutePath());
       }
     }
   }

